sql/core/src/main/scala/org/apache/spark/sql/execution/command/views.scala

```diff
@@ -359,8 +359,13 @@ object ViewHelper {
     "spark.sql.shuffle.",
     "spark.sql.adaptive.")
 
+  private val configAllowList = Seq(
+    SQLConf.DISABLE_HINTS.key
+  )
+
   private def shouldCaptureConfig(key: String): Boolean = {
```

Comment:
could you leave a comment to make the filtering rule clear? i.e. allowList has higher priority than denyList.

Reply:
Added the comment.